backend/app/api/src/seeds/1754560914-groups-prices.ts

```diff
@@ -0,0 +1,270 @@
+import { Migration } from '@simonbackx/simple-database';
+import { Group, OrganizationRegistrationPeriod } from '@stamhoofd/models';
+import { BundleDiscount, BundleDiscountGroupPriceSettings, GroupPrice, GroupPriceDiscount, GroupPriceDiscountType, GroupStatus, OldGroupPrices, ReduceablePrice, TranslatedString } from '@stamhoofd/structures';
+import { Formatter } from '@stamhoofd/utility';
+
+function getParentCategory(group: Group, period: OrganizationRegistrationPeriod) {
+    const parent = group.getParentCategories(period.settings.categories)[0];
+    if (!parent) {
+        return null;
+    }
+    return parent;
+}
+
+async function migratePrices() {
+    for await (const period of OrganizationRegistrationPeriod.select().all()) {
+        // key = group category id
+        const bundleMap = new Map<string, BundleDiscount[]>();
+        const allBundleDiscounts: BundleDiscount[] = [];
+
+        for await (const group of Group.select().where('periodId', period.periodId).andWhere('organizationId', period.organizationId).all()) {
+            // if already migrated
+            if (group.settings.prices.length > 0) {
+                if (group.settings.oldPrices.length > 0) {
+                    group.settings.oldPrices = [];
+                    await group.save();
+                }
+                continue;
+            }
+
+            // if archived
+            if (group.status === GroupStatus.Archived || group.deletedAt !== null) {
+                group.settings.oldPrices = [];
+                group.settings.prices = [
```

Comment:
Denk dat we in het archief wel best ook nog de prijzen gaan behouden, aangezien fiscale attesten er nog vaak aan verbonden zijn